admin_manual/maintenance/migrating.rst

```diff
@@ -62,141 +62,138 @@ When a user tries a URL that is not whitelisted the following error appears:
    :scale: 75%
    :alt: Error message when URL is not whitelisted
 
-=======
-Example
-=======
-
-.. note:: For this example to work, you need this on both servers:
+=================
+Example Migration
+=================
 
+.. note:: For this example to work, you need the following on both servers:
 * Ubuntu 16.04
 * SSH
 * PermitRootLogin set to "yes"
 
-optional:
-
-* Domain name (for Let's Encrypt)
-* Modules: smb-client nfs-common rpcbin (for WND)
-
-
-Install SSH::
-
+Preparation::
+   
+   #     Install ssh
+   #
    apt install ssh -y
-
-Edit ssh-config (enable root ssh login)::
-
+   #
+   #     Edit ssh-config (enable root ssh login)
+   #
    nano /etc/ssh/sshd_config
-
-Change PermitRootLogin to "yes"::
-
+   #
+   #     Change PermitRootLogin to "yes"
+   #
    PermitRootLogin yes
-   
-Restart ssh service::
-
+   #
+   #     Restart ssh service
+   #
    service ssh stop
+   #
    service ssh start
-
-Install ownCloud on new server
+   #
+   #     Install ownCloud on new server
 
 =========
 Migration
 =========
```

Comment:
This shouldn't have both an underline and an overline. That's likely breaking the intended header level.